vertx-lang-groovy-codegen/pom.xml

```diff
@@ -80,12 +85,6 @@
       <scope>test</scope>
     </dependency>
     <!-- Used by failsafe plugin that excludes the compile scope -->
-    <dependency>
```

Comment:
I think there is a good reason we have this dependency can you just upgrade it instead of removing it ?